interface/themes/oemr-bs3-override.css

```diff
@@ -0,0 +1,79 @@
+/*override bootstrap 3 for openemr and some new classes all prefixed with oe-*/
```

Comment:
I think the issue here is that we don't need to enforce pages to display in a consistent way. Flexibility should trump that.
On my testing would recommend placing the following into core.css:
.form-group
.form-horizontal .control-label

Then the following would go into the individual themes:
legend
fieldset
(note that while playing around I removed the background/background-color entries in each of these elements in light style and actually didn't look that bad albeit the shading is more to my taste; this is an example why this should be left to the theme developers)

And the rest seem to all be theme specific (ie. most don't even need to be included in the other themes, but that can be on a case by case basis; for example, come of the oe* stuff may be needed, but I think the button building and button spacing should be left to the theme; and then the responsibility of enforcing rtl for each theme can be done via demand; so this means the current themes will maintain rtl support while your new themes will require some work if a user/developer wishes to pursue this.